package.json

```diff
@@ -134,20 +134,22 @@
     "polyfill-function-prototype-bind": "0.0.1",
     "quick_check": "^0.7.0",
     "raven-js": "^3.12.1",
-    "react": "^15.2.1",
+    "react": "^15.5.4",
     "react-addons-css-transition-group": "^15.3.2",
     "react-addons-perf": "^15.3.0",
     "react-addons-test-utils": "^15.3.0",
```

Comment:
I think I'm going to leave this to another PR (along with the prop types change). It'll touch so many components and I wanted to keep this PR as simple as possible.